File: config-biz/src/main/java/io/gitee/zicai/biz/mapper/PublishedMapper.java
package io.gitee.zicai.biz.mapper;

import io.gitee.zicai.core.entity.Published;
import org.apache.ibatis.annotations.Param;
import tk.mybatis.mapper.common.Mapper;

public interface PublishedMapper extends Mapper<Published> {

    Published getLast(@Param("appName")String appName, @Param("appEnv")String appEnv);

    Published getLastOne(@Param("appId")Long appId);
}

File: config-biz/src/main/java/io/gitee/zicai/biz/mapper/PublishedMapper.kt
package io.gitee.zicai.biz.mapper

import com.baomidou.mybatisplus.core.mapper.BaseMapper
import io.gitee.zicai.core.entity.Published
import org.apache.ibatis.annotations.Param

interface PublishedMapper : BaseMapper<Published> {

    fun getLast(@Param("appName") appName: String, @Param("appEnv") appEnv: String): Published

    fun getLastOne(@Param("appId") appId: Long?): Published
}